storage/migration/runner.go

```diff
@@ -0,0 +1,100 @@
+package migration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/rs/zerolog/log"
+)
+
+var DefaultMigrationConfig = MigrationConfig{
+	BatchByteSize:          32_000_000, // 32 MB
+	ReaderWorkerCount:      2,
+	WriterWorkerCount:      2,
```

Comment:
The worker count doesn't seem to make a big difference, `4` is the same as `2` for both reader and writer.